server/api/auth/login.post.ts

```diff
@@ -4,25 +4,41 @@ import { z } from "zod";
 import { SESSION_COOKIE_NAME, SESSION_STORAGE, SESSION_TTL_SECONDS } from "~/utils/sessionStorage";
 import prisma from "~/utils/db";
 
+// Constants
+const STATUS_CODES = {
+  UNPROCESSABLE_ENTITY: 422,
+  UNAUTHORIZED: 401,
+};
+
+const ERROR_MESSAGES = {
+  VALIDATION_ERROR: "Username and password field is required",
+  USER_NOT_FOUND: "Invalid username or password",
+  INVALID_CREDENTIALS: "Invalid username or password",
+};
+
 const userSchema = z.object({
   username: z.string(),
   password: z.string(),
 });
 
 export default defineEventHandler(async (event) => {
+  // Validate request body
```

Comment:
This comment is unnecessary because it literally repeats the name of the function below.
```suggestion
```